Explore/Logs: Update Parsed fields to Detected fields (#28881)
grafana/grafana: docs/sources/whatsnew/whats-new-in-v6-5.md

{{< docs-imagebox img="/img/docs/v65/cloudwatch-deep-linking.png" max-width="500px" class="docs-image--right" caption="CloudWatch deep linking" >}}

> Left-clicking a time series in the panel displays a context menu with a link to `View in CloudWatch console`. Clicking that link opens the CloudWatch console and displays all the metrics for that query. If you are not currently logged in to the CloudWatch console, then the link opens the login page. The link is valid for any account, but it only displays the right metrics if you are logged in to the account that corresponds to the selected data source in Grafana.

Reviewer: ```suggestion
Click a time series in the panel to display a context menu with a link to **View in CloudWatch console**. Clicking that link opens the CloudWatch console and displays all the metrics for that query. If you are not currently logged in to the CloudWatch console, then the link opens the login page. The link is valid for any account, but it only displays the right metrics if you are logged in to the account that corresponds to the selected data source in Grafana.
```

Metrictank returns 2 kinds of additional metadata along its responses:

- **Performance information:** Time spent querying index, fetching data, running processing functions, the number of series and points fetched, cache hits/misses, etc. This can be useful for optimizing queries or tuning the chunk cache.

> - **Lineage information about the returned series:** Which archive was fetched from (raw or rollup), which (if any) runtime consolidation was applied (using which processing function), etc. This is very useful information for anyone trying to understand how their data was generated and why it may not look as expected.

Reviewer: ```suggestion
- **Lineage information about the returned series:** Which archive was fetched from (raw or rollup), which (if any) runtime consolidation was applied (using which processing function), and so on. This is very useful information for anyone trying to understand how their data was generated and why it might not look as expected.
```

To see the metadata response from Metrictank you can inspect the response using the Query Inspector found in the panel queries tab.

> Grafana 6.5 includes a new `Panel Inspector` in alpha/preview where you also can see the metadata response from Metrictank.

Reviewer: @ivanahuckova , this paragraph looks outdated. Besides the Grafana 6.5 mention, the panel inspector is out of beta and was redone as part of 7.0.

Author: This is **Whats new in 6.5**, not our documentation. I probably opened the file and then saved it, which created formatting changes. But we don't have to update content of this.

Reviewer 2: I am inclined to agree.
